### ISSUE TITLE
Hide the taskbar icon on Windows when minimized

### ISSUE THREAD

**Issue by AngelEzquerra** [twolfson/google-music-electron#16]
It would be nice if it was possible to configure Google Music Electron to hide its taskbar icon when the program is minimized. You could still use the trayicon to show the program window (and its taskbar icon) when needed. This would let you keep google music electron open, playing music, without ocupying precious space on the Windows taskbar.

**Comment by twolfson** [comment]
This can be enabled via the `--minimize-to-tray` flag in the CLI:

https://github.com/twolfson/google-music-electron/tree/1.19.0#cli

``` bash
google-music-electron --minimize-to-tray
```

**Comment by AngelEzquerra** [comment]
Thank you. In the end I used the "--skip-taskbar" flag, which suits me even more. However I found a problem with the "--minimize-to-tray" flag. When you use it, the minimize button on the Google Music Electron window does not work well. It hides the window, as expected, but the taskbar item is still there (i.e. not hidden). If you click on it it will show the main window again. If instead you click on the system tray google music electron icon it will then hide the taskbar button and keep the window hidden. Clickinng on the system tray icon again will show the window and the taskbar item (as expected).

**Comment by twolfson** [comment]
Hmm, strange. Minimizing should work, here's the binding:

https://github.com/twolfson/google-music-electron/blob/1.19.0/lib/google-music-electron.js#L235-L238

What OS are you on so we can attempt to reproduce?

**Comment by AngelEzquerra** [comment]
I am using Windows 7.

**Comment by twolfson** [comment]
Attempting to reproduce this now. I think I am seeing the same behavior with:

``` bash
node bin\google-music-electron.js --minimize-to-tray
```

It looks like the taskbar icon disappears and then re-opens a new window. I am going to see if this is due to the version of `electron` we are on. If not, then we will need to create a standalone gist to reproduce the error and submit a ticket on `electron`.

**Comment by twolfson** [comment]
It looks like upgrading `electron` should resolve the issue. Going to do that and release :tada:

**Comment by twolfson** [comment]
Looked through the CHANGELOG to see what happened. It looks like `0.33.6` was our relevant release:

https://github.com/atom/electron/releases/tag/v0.33.6

**Comment by twolfson** [comment]
The malfunctioning `--minimize-to-tray` has been patched in `1.21.0`.
